server/webapp/WEB-INF/rails.new/app/helpers/about_helper.rb

```diff
@@ -0,0 +1,69 @@
+##########################################################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module AboutHelper
+  def jvm_version
+    system_property('java.version')
+  end
+
+  def os_information
+    [system_property('os.name'), system_property('os.version')].join(' ')
+  end
+
+  def system_property(property)
+    system_environment.getPropertyImpl(property)
+  end
+
+  def available_space
+    artifact_dir = Spring.bean(com.thoughtworks.go.server.service.ArtifactsDirHolder.java_class).getArtifactsDir
```

Comment:
I think you should just put ```:artifacts_dir_holder``` in [services.rb](https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails.new/lib/services.rb#L29) and use that. That's what it is for.